src/core/isEmpty.js

```diff
@@ -1,8 +1,14 @@
 /** @license ISC License (c) copyright 2016 original and current authors */
 /** @author Ian Hofmann-Hicks (evil) */
 const isObject = require('./isObject')
+const isMonoid = require('./isMonoid')
+const equals = require('./equals')
 
 function isEmpty(x) {
+  if(isMonoid(x)) {
+    return equals(x, x.empty())
```

Comment:
to make this compatible with FL as well as crocks types, we should grab the empty off of the constructor doing `x.constructor.empty()`.